### ISSUE TITLE
Cannot login with apiToken

### ISSUE THREAD

**Issue by defyingphysics** [successible/cleanslate#176]
Hey, 

I have managed to get the app hosted locally on a Raspberry Pi by setting up the NEXT_PUBLIC_HASURA_DOMAIN and the domain in the caddy file to my reserved RPI IP e.g. https://192.168.0.4:4000. 

I managed to login to the Hasura Console and made a new user in profiles. When I try to use apiToken at 192.168.0.4:4000,  I get the error 
"No matching profile found! Make sure to login with apiToken, not authId."

I checked the docker postgres container running on the PI and the user I created is indeed in the profiles table. 

I am out of ideas as I have now followed steps 5 and 6 from the [hosting instructions](https://github.com/successible/cleanslate?tab=readme-ov-file#how-do-i-host-clean-slate) a few times with no success. 

Here are the docker containers running on my RPI. 

  cleanslate-authentication-server                 "docker-entrypoint.s…"   20 minutes ago   Up 20 minutes             0.0.0.0:3001->3001/tcp, :::3001->3001/tcp   authentication-server
  cleanslate-client                                "busybox httpd -f -v…"   20 minutes ago   Up 20 minutes             0.0.0.0:3000->3000/tcp, :::3000->3000/tcp   client
 hasura/graphql-engine:latest.cli-migrations-v3   "docker-entrypoint.s…"   20 minutes ago   Up 20 minutes (healthy)   0.0.0.0:8080->8080/tcp, :::8080->8080/tcp   graphql-server
   postgres:15                                      "docker-entrypoint.s…"   20 minutes ago   Up 20 minutes (healthy)   5432/tcp                                    database

**Comment by paulzakin** [comment]
Unfortunately, that error is not really an error. It is just saying that the apiToken for the profile does not match the one you passed in on the login screen. Are you sure you are using apiToken, not authId? And you are sure you are passing the value shown in the console? 

Also, that port seems a bit odd. Almost everyone who runs the app does so on :443 like a standard https://example.com sort of thing. Perhaps that is the problem? I have never tested having the proxy for the app on a non-standard port like that. Perhaps switching to a different proxy (like nginx) or serving it at :443 will help. For example, even when testing locally, I run it at https://localhost. 

Beyond that, the check is just returning nothing for this GraphQL request. So if this query works in the Hasura Console ($token is a variable) it might a problem with some other part of your setup. My instinct says it is the proxy.

query GET_PROFILES($token: uuid!) {
      profiles(
        where: {
          _or: [{ apiToken: { _eq: $token } }]
        }
      ) {
        authId
        id
      }
    }

Beyond that, my ability to debug here is pretty limited :( 
Every self hosting set up is quite different, no one else has reported this error,  and I cannot replicate this locally. I am sorry I cannot be of more help. If you cannot get it to work, you can of course always use the instance we host (app.cleanslate.sh), but totally understand if that is not an option

**Comment by defyingphysics** [comment]
Hey, 

Thanks for the fast response. I am a bit of a noob when it comes to Fullstack stuff (Mechatronics Eng by degree so the code I write is in C++ and C lol) learning networking as I go as I think a home lab is a perfect place to do it. 

100% I am using apiToken see attached image to check that it is in the right spot. I also confirmed that the PostGres DB on the Pi has the profile that I created on my PC. So at least I was able to access the console and it made the changes on the PI from my PC.
![pic1](https://github.com/user-attachments/assets/06559399-6b84-4f6e-8038-61dcc4af32db)

I chose the 4000 arbitrarily as a starting point. I tried LocalHost, and whilst I can curl https://localhost locally on the PI, and get what seems to be the homepage, I am running the PI headless so need to try access the app on another PC connected to my network.  When I try https://192.168.0.4/ all I get is SSL_ERROR_INTERNAL_ERROR_ALERT. 

That's where the idea to have the my IP plus an arbitrary port came up. Tried to replicate what Apps running on my TrueNas Scale server were doing. It seemed to work till I hit this last wall. 

I saw that the example mentioned https://example.com/ but for that I assume I need to purchase/acquire a domain, something which I am a whole lot of knowledge away from doing I feel haha. 

Seems I probably need to go and learn a lot more about what is actually going on with proxies and related things. Last night I was googling what Hasura even is and so on haha. I am confused for example how it can run on a server and compete with other apps without using a unique port, I guess it has to do with SubDomains? 

Finally, I tried the query you sent on the Hasura Console and it seems to work, see  below ![pic2](https://github.com/user-attachments/assets/ca11db02-2a6f-457a-ae91-dace96474b72)


In the mean time I should probably try out the app you hosted and see how it is :) Then perhaps I can come back to the self-hosting experiment when I have more knowledge... 

By the way, really liked your ethos on "We felt other apps were too complicated and promoted perfectionism. We just wanted one that promoted self-compassion and education. That would not make you feel terrible for messing up!"

**Comment by paulzakin** [comment]
Yup, the best part about self-hosting is you can do it on your own time and learn a lot :) But it can be hard to learn everything at once.

I will close this issue, and yup, please do use the main website and give it a shot!

**Comment by jonneolie** [comment]
Hi @paulzakin
I'm actually having the same issue.
I can check the profile from the request in hasura, with the same query as @defyingphysics, but I also can't login.

I'm using nginx, and for the rest I've set up the environment exactly like stated in the README.
Except I changed all the ports to custom ports, that was more convenient for me, cause they were in use already.

I do have my own domain, so I use something like https://cleanslate.example.com.
In the NGINX logs I see the POST of the login, but I don't see any log activity in the containers.

I'm no beginner, so I tried all troubleshooting that I know of, but I can't get to login.

**Comment by paulzakin** [comment]
Can you try it with not custom ports? It may be required to use those exact ports? basically, the way the self hosted login system works is the authentication server makes a request to the API server (Hasura) along that URL (graphql-server) is the Docker hostname. So my hunch is when the ports are different stuff can go wrong. So this will be a good chance to see if we can isolate it together.

http://graphql-server:${hasuraPort}/v1/graphql


Can you share your nginx config? That might help as well.

**Comment by jonneolie** [comment]
Okay, I just tried this and the funny thing is, I can login now, but it hangs on a screen with the purple ...'s.

![image](https://github.com/user-attachments/assets/825e8fa0-3995-49a7-a113-2996378c5e0f)

I tried to create a new profile too, that does the same, it logs in, but then hangs. 

`
server {
        listen 443;
        server_name cleanslate.myhiddendomain.com;
        ssl_certificate     /etc/nginx/ssl/cleanslate.myhiddendomain.com.crt;
        ssl_certificate_key /etc/nginx/ssl/cleanslate.myhiddendomain.com.key;

      # HTTP Security Headers
      add_header Referrer-Policy "strict-origin";
      add_header Strict-Transport-Security "max-age=31536000; includeSubDomains;";
      add_header X-Content-Type-Options "nosniff";
      add_header X-Frame-Options "DENY";
      add_header X-XSS-Protection "1; mode=block;";
      # You can remove the Google, Firebase, and Sentry policies if you are not using them

      location /v1 {
        # API (Hasura)
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection 'Upgrade';
        proxy_set_header Host $host;
        proxy_pass http://localhost:8080;
      }

      location /v2 {
          # API (Hasura)
          proxy_pass http://localhost:8080;
      }

      location /console {
          # Admin panel (Hasura)
          proxy_pass http://localhost:8080;
          add_header Content-Security-Policy "";
      }

      location /auth {
          # Authentication server (Express.js)
          proxy_pass http://localhost:3001;
      }

      location /healthz {
          # Health check (Hasura)
          proxy_pass http://localhost:8080;
      }

      location / {
          # Static files (Clean Slate)
          proxy_pass http://localhost:3000;
      }

  }
`

What I had:
`
server {
        listen 443;
        server_name cleanslate.myhiddendomain.com;
        ssl_certificate     /etc/nginx/ssl/cleanslate.myhiddendomain.com.crt;
        ssl_certificate_key /etc/nginx/ssl/cleanslate.myhiddendomain.com.key;

      # HTTP Security Headers
      add_header Referrer-Policy "strict-origin";
      add_header Strict-Transport-Security "max-age=31536000; includeSubDomains;";
      add_header X-Content-Type-Options "nosniff";
      add_header X-Frame-Options "DENY";
      add_header X-XSS-Protection "1; mode=block;";
      # You can remove the Google, Firebase, and Sentry policies if you are not using them
   

  location /v1 {
        # API (Hasura)
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection 'Upgrade';
        proxy_set_header Host $host;
        proxy_pass http://localhost:8793;
      }

      location /v2 {
          # API (Hasura)
          proxy_pass http://localhost:8793;
      }

      location /console {
          # Admin panel (Hasura)
          proxy_pass http://localhost:8793;
          add_header Content-Security-Policy "";
      }

      location /auth {
          # Authentication server (Express.js)
          proxy_pass http://localhost:8794;
      }

      location /healthz {
          # Health check (Hasura)
          proxy_pass http://localhost:8793;
      }

      location / {
          # Static files (Clean Slate)
          proxy_pass http://localhost:8792;
      }
`

I tried to fix the layout.. But somehow it keeps messing it up. If you want me to add it to a file, and attach it, I can.
Basically, i used your nginx conf of the README, and removed the two headers.

Also, I couldn't start nging, cause of:

      location /auth {
          # Authentication server (Express.js)
          proxy_pass localhost:3001;
      }
      
      This was missing (I THINK) http://
      
      This could be another issue though, that I may have messed up something.

**Comment by paulzakin** [comment]
Ok, so in order. 

1. I think you need the http://. However, we should see if we can get your original nginx configuration to work. If not, yup - you should try to get our "official" configuration to work.

2. I updated the documentation with a note about custom ports and how to handle it. You basically did not specify HASURA_PORT, but that is not your fault, it was a bit of a hidden feature. So re-read #2 and #3 of the How to Deploy documentation and try again.

3. Clear your cache, cookies, and localStorage in the brower. That should resolve the logged in page but empty error.

**Comment by jonneolie** [comment]
Oh, I should have mentioned it before, but I had added the following to my .env file to change the ports.
CLIENT_PORT=8792
HASURA_PORT=8793
AUTHENTICATION_SERVER_PORT=8794

So I already had those changed, matching with the docker-compose file and the nginx conf. So this won't work unfortunately.

**Comment by paulzakin** [comment]
Ok, @jonneolie I replicated and patched in #185 - give it a shot!

**Comment by jonneolie** [comment]
@paulzakin It didn't work for me, but I do get an error in the graph container now when I login with the token!


`
{"detail":{"http_info":{"content_encoding":null,"http_version":"HTTP/1.1","ip":"127.0.0.1","method":"GET","status":200,"url":"/healthz"},"operation":{"request_id":"f287cb85-b175-4311-ba98-ad47a44adca6","request_mode":"non-graphql","response_size":2,"uncompressed_response_size":2},"request_id":"f287cb85-b175-4311-ba98-ad47a44adca6"},"level":"info","timestamp":"2024-10-01T08:10:44.910+0000","type":"http-log"}
{"detail":{"generated_sql":{"profiles":{"prepared_arguments":[[2,"ae99ab9a-f5fa-4136-83ab-cf2332af75c1"]],"query":"SELECT  coalesce(json_agg(\"root\" ), '[]' ) AS \"root\" FROM  (SELECT  row_to_json((SELECT  \"_e\"  FROM  (SELECT  \"_root.base\".\"authId\" AS \"authId\", \"_root.base\".\"id\" AS \"id\"       ) AS \"_e\"      ) ) AS \"root\" FROM  (SELECT  *  FROM \"public\".\"profiles\"  WHERE ((\"public\".\"profiles\".\"apiToken\") = (($2)::uuid))     ) AS \"_root.base\"      ) AS \"_root\"      "}},"kind":"database","query":{"operationName":"GET_PROFILES","query":"\n    query GET_PROFILES($token: uuid!) {\n      profiles(\n        where: {\n          _or: [{ apiToken: { _eq: $token } }]\n        }\n      ) {\n        authId\n        id\n      }\n    }\n  ","variables":{"token":"ae99ab9a-f5fa-4136-83ab-cf2332af75c1"}},"request_id":"7aead6f2-b21b-4d4f-b7dd-e7597fd13427"},"level":"info","span_id":"8a4b1daa3853bbcf","timestamp":"2024-10-01T08:11:07.935+0000","trace_id":"2086afe3f633af4ab0a271cfde9c5b64","type":"query-log"}
{"detail":{"http_info":{"content_encoding":null,"http_version":"HTTP/1.1","ip":"172.21.0.5","method":"POST","status":200,"url":"/v1/graphql"},"operation":{"query":{"operationName":"GET_PROFILES","query":"\n    query GET_PROFILES($token: uuid!) {\n      profiles(\n        where: {\n          _or: [{ apiToken: { _eq: $token } }]\n        }\n      ) {\n        authId\n        id\n      }\n    }\n  ","variables":{"token":"ae99ab9a-f5fa-4136-83ab-cf2332af75c1"}},"query_execution_time":1.541995e-3,"request_id":"7aead6f2-b21b-4d4f-b7dd-e7597fd13427","request_mode":"single","request_read_time":1.944e-6,"response_size":117,"uncompressed_response_size":117,"user_vars":{"x-hasura-role":"admin"}},"request_id":"7aead6f2-b21b-4d4f-b7dd-e7597fd13427"},"level":"info","span_id":"8a4b1daa3853bbcf","timestamp":"2024-10-01T08:11:07.935+0000","trace_id":"2086afe3f633af4ab0a271cfde9c5b64","type":"http-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"2d5ad745-4911-4dbf-a683-b5cf8eba0645"},"event":{"type":"accepted"},"user_vars":null},"level":"info","timestamp":"2024-10-01T08:11:07.935+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"2d5ad745-4911-4dbf-a683-b5cf8eba0645"},"event":{"type":"closed"},"user_vars":null},"level":"info","timestamp":"2024-10-01T08:11:07.935+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"2d5ad745-4911-4dbf-a683-b5cf8eba0645"},"event":{"detail":"Could not verify JWT: JWSError JWSInvalidSignature","type":"conn_err"},"user_vars":null},"level":"error","timestamp":"2024-10-01T08:11:07.935+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"2d5ad745-4911-4dbf-a683-b5cf8eba0645"},"event":{"type":"closed"},"user_vars":null},"level":"info","timestamp":"2024-10-01T08:11:07.935+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"2e2b8ee8-6bb0-4a18-b1e4-e8173945c7b9"},"event":{"type":"accepted"},"user_vars":null},"level":"info","timestamp":"2024-10-01T08:11:10.939+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"2e2b8ee8-6bb0-4a18-b1e4-e8173945c7b9"},"event":{"type":"closed"},"user_vars":null},"level":"info","timestamp":"2024-10-01T08:11:10.939+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"2e2b8ee8-6bb0-4a18-b1e4-e8173945c7b9"},"event":{"detail":"Could not verify JWT: JWSError JWSInvalidSignature","type":"conn_err"},"user_vars":null},"level":"error","timestamp":"2024-10-01T08:11:10.939+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"2e2b8ee8-6bb0-4a18-b1e4-e8173945c7b9"},"event":{"type":"closed"},"user_vars":null},"level":"info","timestamp":"2024-10-01T08:11:10.939+0000","type":"websocket-log"}
{"detail":{"http_info":{"content_encoding":null,"http_version":"HTTP/1.1","ip":"127.0.0.1","method":"GET","status":200,"url":"/healthz"},"operation":{"request_id":"d400e6b0-ef0e-42ca-a804-aed34c013e2e","request_mode":"non-graphql","response_size":2,"uncompressed_response_size":2},"request_id":"d400e6b0-ef0e-42ca-a804-aed34c013e2e"},"level":"info","timestamp":"2024-10-01T08:11:14.943+0000","type":"http-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"ce692e40-6eb9-4b5a-80ad-7193b07373e5"},"event":{"type":"accepted"},"user_vars":null},"level":"info","timestamp":"2024-10-01T08:11:16.946+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"ce692e40-6eb9-4b5a-80ad-7193b07373e5"},"event":{"detail":"Could not verify JWT: JWSError JWSInvalidSignature","type":"conn_err"},"user_vars":null},"level":"error","timestamp":"2024-10-01T08:11:16.946+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"ce692e40-6eb9-4b5a-80ad-7193b07373e5"},"event":{"type":"closed"},"user_vars":null},"level":"info","timestamp":"2024-10-01T08:11:16.946+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"ce692e40-6eb9-4b5a-80ad-7193b07373e5"},"event":{"type":"closed"},"user_vars":null},"level":"info","timestamp":"2024-10-01T08:11:16.946+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"5498c6ce-aa9c-4f97-a6ba-26bce6a5e1d9"},"event":{"type":"accepted"},"user_vars":null},"level":"info","timestamp":"2024-10-01T08:11:23.953+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"5498c6ce-aa9c-4f97-a6ba-26bce6a5e1d9"},"event":{"detail":"Could not verify JWT: JWSError JWSInvalidSignature","type":"conn_err"},"user_vars":null},"level":"error","timestamp":"2024-10-01T08:11:23.953+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"5498c6ce-aa9c-4f97-a6ba-26bce6a5e1d9"},"event":{"type":"closed"},"user_vars":null},"level":"info","timestamp":"2024-10-01T08:11:23.953+0000","type":"websocket-log"}
{"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"5498c6ce-aa9c-4f97-a6ba-26bce6a5e1d9"},"event":{"type":"closed"},"user_vars":null},"level":"info","timestamp":"2024-10-01T08:11:23.953+0000","type":"websocket-log"}
`

Hope this helps!

**Comment by paulzakin** [comment]
Something is wrong with your JWT. See the error: "Could not verify JWT: JWSError JWSInvalidSignature". That is the clue! 

Make sure that the `<second-long-secret-value>` set in the .env has been replaced with a very long, random string. And that the two match. And make sure to try from a Incognito window.

```
HASURA_GRAPHQL_JWT_SECRET='{"type":"HS256","key":"<second-long-secret-value>"}'
JWT_SIGNING_SECRET=<second-long-secret-value>'
```

I updated the documentation to make the entire .env requirements more explict - please give it a read again and see if that helps.

**Comment by paulzakin** [comment]
Going to close this pending further input. Feel free to reopen!

**Comment by trenchcoatjedi** [comment]
Same error. Cloned repo today. Used the exact .env and caddyfile examples from the readme just changing the domain, passwords and secrets. Can access app, auth console and Hasura DB. Created multiple profiles but the app gives the "No matching profile found! Make sure to login with apiToken, not authId." error no matter which apiToken I try.

**Comment by paulzakin** [comment]
Ok @trenchcoatjedi and @jonneolie - give the README.md another go. I added in a script to autogenerate the .env and Caddyfile, which should hopefully eliminate the error you got. I think it was because the README.md was incorrect, and HASURA_GRAPHQL_JWT_SECRET and JWT_SIGNING_SECRET did not have the same key.

**Comment by trenchcoatjedi** [comment]
Thanks for looking into this. Please add `sudo apt install uuid-runtime ` to the instructions for debian/ubuntu distributions as this is required to execute configuration.sh (will fail otherwise) and this package is not part of some versions of the distribution. 

Of note, configuration.sh generated identical files (except for passwords) to what I had been using before. Also, make note in the documentation that you must own a FQDN and have port 80 forwarded so caddy will succeed in letsencrypt certificate generation.

I'm using Ubuntu 24.10 and I've managed to get to the same place as this guy where it's able to login but hanging on purple loading circles:

> Okay, I just tried this and the funny thing is, I can login now, but it hangs on a screen with the purple ...'s.
> 
> ![image](https://github.com/user-attachments/assets/825e8fa0-3995-49a7-a113-2996378c5e0f)
>

**Comment by paulzakin** [comment]
Ok, I added stuff to the README.md - I think I need more logging information to really help you debug, either from the browser or the server. Also, can you try totally clearing the cache and the service worker - you can do the latter with the "clear site data" from the application tab. 

<img width="1470" height="956" alt="Image" src="https://github.com/user-attachments/assets/9c3a93f1-ded2-4127-ba31-f2e016047486" />

**Comment by trenchcoatjedi** [comment]
It seems to be a websockets issue:

```
graphql-server         | {"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"38e4bcfe-0436-42d9-945a-7b9572efaea7"},"event":{"detail":{"code":"access-denied","error":"received origin header does not match configured CORS domains","path":"$"},"type":"rejected"},"user_vars":null},"level":"error","timestamp":"2025-07-05T13:33:36.763+0000","type":"websocket-log"}
graphql-server         | {"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"656578d1-9720-4589-ab25-076d222cf1f6"},"event":{"detail":{"code":"access-denied","error":"received origin header does not match configured CORS domains","path":"$"},"type":"rejected"},"user_vars":null},"level":"error","timestamp":"2025-07-05T13:33:42.769+0000","type":"websocket-log"}
graphql-server         | {"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"de6ff6de-f934-453d-9cba-2fd933789608"},"event":{"detail":{"code":"access-denied","error":"received origin header does not match configured CORS domains","path":"$"},"type":"rejected"},"user_vars":null},"level":"error","timestamp":"2025-07-05T13:33:51.779+0000","type":"websocket-log"}
graphql-server         | {"detail":{"http_info":{"content_encoding":null,"http_version":"HTTP/1.1","ip":"127.0.0.1","method":"GET","status":200,"url":"/healthz"},"operation":{"request_id":"cdae22db-7b32-44ec-8130-271057bdb276","request_mode":"non-graphql","response_size":2,"uncompressed_response_size":2},"request_id":"cdae22db-7b32-44ec-8130-271057bdb276"},"level":"info","timestamp":"2025-07-05T13:34:01.792+0000","type":"http-log"}
graphql-server         | {"detail":{"connection_info":{"msg":null,"token_expiry":null,"websocket_id":"1a150ec7-718a-464a-871b-4856329ce046"},"event":{"detail":{"code":"access-denied","error":"received origin header does not match configured CORS domains","path":"$"},"type":"rejected"},"user_vars":null},"level":"error","timestamp":"2025-07-05T13:34:08.801+0000","type":"websocket-log"}
```

<img width="1903" height="656" alt="Image" src="https://github.com/user-attachments/assets/9313e4a3-19e0-42ea-b07d-1f799174655a" />

**Comment by paulzakin** [comment]
Looks like it is a CORS issue - what is the origin being sent - the value of the CORS is set by the domain in the .env which is then used in the docker compose file

**Comment by trenchcoatjedi** [comment]
The .env has https://my.fqdn.com as the domain. This is the same place the app is hosted and accessible. Is the syntax of the .env correct?

```
root@host:/opt/cleanslate# cat .env
AUTHENTICATION_SERVER_PORT=3001
CLIENT_PORT=3000
HASURA_PORT=8080
POSTGRES_PORT=5432
NEXT_PUBLIC_HASURA_DOMAIN=https://my.fqdn.com
HASURA_GRAPHQL_ADMIN_SECRET=super-secret-1
HASURA_GRAPHQL_JWT_SECRET='{"type":"HS256","key":"super-secret2"}'
JWT_SIGNING_SECRET=super-secret2
POSTGRES_PASSWORD=super-secret3
```

**Comment by paulzakin** [comment]
Ah, no - you cannot have https:// - it is just the domain without the protocol - the value should just be `my.fqdn.com` - I have updated the generation script accordingly.

**Comment by trenchcoatjedi** [comment]
Works now, thanks!

**Comment by paulzakin** [comment]
Awesome - hope you like it!
